desktop/plugins/network/index.tsx

```diff
@@ -206,14 +208,18 @@ export default class extends FlipperPlugin<State, any, PersistedState> {
   }
 
   init() {
-    this.client.supportsMethod('mockResponses').then((result) =>
+    this.client.supportsMethod('mockResponses').then((result) => {
+      const routes = JSON.parse(
+        localStorage.getItem(LOCALSTORAGE_MOCK_ROUTE_LIST_KEY) || '{}',
```

Comment:
You should use `persistentState` and `setPersistentState` from props. This can preserve the change for each session.

Reply:
I use `localStorage` to persis the mocks, so they can survive even if the Flipper app is restarted

Reply:
Then, I'll let @mweststrate stamp.

Reply:
Note that using `localStorage` means that all instances across different devices / applications will have the same mocks available (which can be seen both as a feature and a bug I guess :-P)